### PR TITLE
Add P-384 combined with ML-KEM-1024

### PR DESCRIPTION
**Pull request by tomato42**

### CODE REVIEW

File: draft-kwiatkowski-tls-ecdhe-mlkem.md [hunk 1]
```diff
@@ -157,7 +183,7 @@ especially those that can be applied by remote attackers.
 
 # IANA Considerations
 
-This document requests/registers two new entries to the TLS Supported Groups registry, according
+This document requests/registers four new entries to the TLS Supported Groups registry, according
```
Comment (kriskwiatkowski) [inline draft-kwiatkowski-tls-ecdhe-mlkem.md:186]: three

**Review by kriskwiatkowski — COMMENTED** [review]
First pass

File: draft-kwiatkowski-tls-ecdhe-mlkem.md (hunk 1 above)
Comment (tomato42) [reply draft-kwiatkowski-tls-ecdhe-mlkem.md]: right, fixed.

**Comment by kriskwiatkowski** [comment]
Thanks, I suggest we wait with IANA request until it we know if code point should be "recommended" or not.

**Pull request merged**